websites/K/Kinopoisk/presence.ts

```diff
@@ -0,0 +1,215 @@
+const presence = new Presence({
+	clientId: "1034799018980679680",
+});
+
+let video = {
+	duration: 0,
+	currentTime: 0,
+	paused: true,
+};
+
+presence.on(
+	"iFrameData",
+	(data: { duration: number; currentTime: number; paused: boolean }) => {
+		video = data;
+	}
+);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			details: "Где-то на сайте",
+			smallImageText: "Kinopoisk",
+		},
+		[privacy, time] = await Promise.all([
+			presence.getSetting<boolean>("privacy"),
+			presence.getSetting<boolean>("time"),
+		]),
+		{ hostname, pathname } = document.location,
+		textContent = (tags: string) => {
+			return document.querySelector(tags)?.textContent?.trim();
+		},
+		pageTitle = (string: string) => {
+			return document
+				.querySelector<HTMLTitleElement>("title")
+				.textContent.split(string);
+		},
+		strings = await presence.getStrings({
+			play: "general.watchingVid",
+			pause: "general.paused",
+		});
+
+	switch (hostname) {
+		case "www.kinopoisk.ru":
+			presenceData.largeImageKey = "https://i.imgur.com/FzNE1zD.png";
```

Comment:
If you would like, you can store all the Assets (`smallImageKey` and `largeImageKey`) in a `Enum` instead so future maintenance is simple
https://github.com/PreMiD/Presences/blob/0acfde95e75d8c8a2bcdbbdc2482a4cb4e9e5705/websites/T/Toyhouse/presence.ts#L10-L13